cli/azd/test/functional/aspire_test.go

```diff
@@ -43,7 +44,23 @@ func restoreDotnetWorkload(t *testing.T) {
 
 		wr := logWriter{initialTime: time.Now(), t: t, prefix: "restore: "}
 		commandRunner := exec.NewCommandRunner(nil)
-		runArgs := newRunArgs("dotnet", "workload", "restore", "--skip-sign-check").WithCwd(appHostProject).WithStdOut(&wr)
+		cmd := "dotnet"
+		args := []string{"workload", "restore", "--skip-sign-check"}
+
+		// On platforms where the system requires `sudo` to install workloads (e.g. macOS and Linux when using system wide
```

Comment:
@vhvb1989 I think I mentioned to you at one point I had configured my machine in a way such that I could just add `sudo` here and then `go test` could run these tests even with the version of `dotnet` I have installed which is system wide and requires `sudo` for dotnet commands.

I decided I would just add this technology to the test itself, so I the future I can just set `AZD_TEST_DOTNET_WORKLOAD_USE_SUDO=1` (likely in my `.profile`) and then not worry about editing this file to add `sudo` and then reverting my changes.

You might find this useful as well, since I think you were in the same boat as me, but on Linux.  Perhaps this will also work for you (you'll have to tweak the line in `/etc/sudoers`, of course).